scripts/data/experiment.json

```diff
@@ -156,4 +156,26 @@
     "endDate": "2016-02-02",
     "lastEdited": "2016-02-04",
     "structure": []
+}, {
+    "id": "test9",
+    "title": "Lookit Test",
+    "description": "It's a test",
+    "state": "Active",
+    "eligibilityCriteria": "age >= 3 years and age < 6 years",
+    "structure": [
+        {
+            "type": "lookit-overview",
+            "title": "Learning from Others: overview",
+            "duration": "Ten minutes",
+            "whatHappens": "First your child will see two actors naming familiar objects. Sometimes they're correct, sometimes they're not. Then we'll show the same actors naming unfamiliar objects and ask your child some questions about which answers he or she trusts.",
```

Comment:
If we would like this to be a generic page, I recommend making the schema field names a bit more generic. Names like "whatHappens" don't make sense without looking at the template code (which the user doesn't really want to do), and may not map to other intended uses of the template in the future.

For example, this could be billed as a list of bulleted items: `[{summary: {'title': 'What happens', description: 'First your child...'}}`. If you really wanted to enforce design, you could even limit the number of "overview section" items in the array (no more than 4-5, say)